files-csharp/FastestWayToReadATextFileInCsharp/FastestWayToReadATextFileInCsharp/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using BenchmarkDotNet.Running;
+
+namespace FastestWayToReadATextFileInCsharp
```

Comment:
We can change this file to use top level statements which will greatly simplify this file.